controller.py

```diff
@@ -9,80 +9,107 @@ def __init__(self,path):
+        self.options_appdetail = {"a":self.show_applicants_list,
+                                   "e":self.edit_appscore_start,
+                                   "b":self.show_boot_message,
+                                   "O":self.open_applicant_pdf}
 
-    def show_boot_message(self):
+    def show_boot_message(self,k=None):
         """Shortlist overview"""
         self.view.boot_message(self.path,len(self.shortlist.applicants))
-        self.current_view = "home"
+        self.options = self.options_home
 
-    def show_criteria(self):
+    def show_criteria(self,k=None):
         """Display criteria information"""
         self.view.view_criteria(self.shortlist.role,self.shortlist.role.criteria)
-        self.current_view = "criteria"
 
-    def show_role_info(self):
+    def show_role_info(self,k=None):
         """Display role information"""
         self.view.view_role(self.shortlist.role)
-        self.current_view = "role"
 
-    def show_applicants_list(self):
+    def show_applicants_list(self,k=None):
         """List all applicants"""
         self.view.view_applicants_list(self.shortlist)
-        self.current_view = "applicants_list"
+        self.options = self.options_applist
 
-    def show_applicant_details(self):
+    def show_applicant_details(self,k=None):
         """Select an applicant and view details"""
         try:
             i = int(input("Please enter the applicant number:"))
 
             self.current_applicant = self.shortlist.applicants[i-1]
             self.view.view_applicant_details(self.current_applicant)
-            self.current_view = "applicant_details"
+            self.options = self.options_appdetail
         except (ValueError, IndexError):
             pass
 
-    def open_applicant_pdf(self):
+    def open_applicant_pdf(self,k=None):
         """Open current applicant's CV"""
         startfile(self.current_applicant.cv)
 
+    def edit_appscore_start(self,k=None):
+        """select a criteria to edit score for"""
+        self.view.view_criteria(self.shortlist.role,self.shortlist.role.criteria)
+        options = [str(i) for i in range(len(self.shortlist.role.criteria))]
+        self.options = {i:self.edit_criteria_select for i in options} #generate options based on how many criteria there are
+        #if a criteria number is selected, call the next function
+
+    def edit_criteria_select(self, k=None):
+        self.current_criterion = self.shortlist.role.criteria[int(k)]
+        print(f"You selected {self.current_criterion.name}. Select the score you want to edit")
+        options = [str(i) for i in range(len(self.current_criterion.scores))]
+        self.options = {i:self.edit_score_select for i in options}
+        
+        for index,score in enumerate(self.current_criterion.scores):
+                print(f"{index}: {score}")
+        
+        self.current_score = self.current_criterion.scores[int(k)]
+
+    def edit_score_select(self, k=None):
```

Comment:
We don't need to confirm step. Simply save the new score because, hopefully, it will be quick to change the score if one wants to.